static/js/loading.js

```diff
@@ -28,6 +28,8 @@ exports.make_indicator = function (outer_container, opts) {
         container = inner_container;
     }
 
+    notifications.hide_history_limit_message();
```

Comment:
I don't think we want to hide this in every caller of `loading.make_indicator`.  Why did you add this here?  It seems like you already have something in e.g. `maybe_load_older_messages`